tfhe/docs/core_crypto/tutorial.md

```diff
@@ -9,7 +9,7 @@ Welcome to this tutorial about `TFHE-rs` `core_crypto` module.
 To use `TFHE-rs`, it first has to be added as a dependency in the `Cargo.toml`:
 
 ```toml
-tfhe = { version = "0.4.1", features = [ "x86_64-unix" ] }
+tfhe = { version = "0.4.2", features = [ "x86_64-unix" ] }
```

Comment:
I assume it's a backport. This would be changed for `"0.5.0"` for the new release ?

Reply:
My bad, I didn't read the content of the PR :zany_face: